Documentation/guides/messages/xa1023.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Xamarin.Android warning XA1023
+description: XA1023 warning code
+ms.date: 05/18/2020
+---
+# Xamarin.Android warning XA1023
+
+## Example messages
+
+```
+warning XA1023: Using the DX Dex Compiler is deprecated. Please update `$(AndroidDexTool)` to `d8`.
+```
+
+## Issue
+
+Google has deprecated the DX Dex Compiler in favor of the [D8 Dex
+Compiler][d8]. On [February 1, 2021][dx], DX will no longer be a part
+of Android SDK or Android Studio.
+
+[d8]: https://developer.android.com/studio/command-line/d8
+[dx]: https://android-developers.googleblog.com/2020/02/the-path-to-dx-deprecation.html
+
+## Solution
+
+Update the `$(AndroidDexTool)` MSBuild property to `d8` to select the
```

Comment:
We should include instructions on how to do this in the IDEs? I can imagine most developers will be using the IDE's not editing csproj by hand.

Reply:
👍 I used what we had in `xa1011.md` as an example.